content/docs/hello-world.md

```diff
@@ -17,6 +18,9 @@ ReactDOM.render(
 
 It displays a heading saying "Hello, world!" on the page.
 
+
+TODO - This link does not work, it opens a blank code pen project.
```

Comment:
The empty link is being addressed by https://github.com/reactjs/reactjs.org/pull/4232 PR